diff -Nur wmsvencd-0.5.0/Makefile wmsvencd-0.5.0-p/Makefile
--- wmsvencd-0.5.0/Makefile	Sat May  8 09:54:08 1999
+++ wmsvencd-0.5.0-p/Makefile	Wed Jul 21 16:38:28 1999
@@ -1,11 +1,13 @@
 BINDIR = /usr/local/bin
 MANDIR = /usr/local/man/man1
 CFLAGS = -O6 -fno-strength-reduce
+LIBDIR = -L/usr/X11R6/lib
+LIBS = -lXpm -lXext -lX11
 
 all: wmsvencd
 
 wmsvencd: cdctl.o wmsvencd.o
-	c++ -o wmsvencd $(CFLAGS) -L/usr/X11R6/lib wmsvencd.o cdctl.o -lXpm -lXext -lX11
+	c++ -o wmsvencd $(CFLAGS) $(LIBDIR) wmsvencd.o cdctl.o $(LIBS)
 
 wmsvencd.o: wmsvencd.cc
 	c++ $(CFLAGS) -I/usr/X11R6/include -I/include -c wmsvencd.cc -o wmsvencd.o
@@ -21,4 +23,4 @@
 install:
 	mkdir -p $(BINDIR)
 	install -c -s -m 0755 wmsvencd $(BINDIR)
-	install -c -m 0644 wmsvencd.1x $(MANDIR)
+	install -c -m 0644 wmsvencd.1x $(MANDIR)
\ No newline at end of file
